src/ui/widgets/Label/label.tsx

```diff
@@ -55,10 +55,17 @@ export const LabelComponent = (
   if (transparent) {
     style["backgroundColor"] = "transparent";
   }
+  if (props.visible) {
```

Comment:
We're kind of duplicating `useCommonCss()` here, if it's not working there we should change it slightly.

Reply:
That is a fair point, I will change it to use it.